src/main/scripts/addon.bat

```diff
@@ -108,6 +108,7 @@ move /y "%PLF_HOME%\addons\addons-manager.jar.new" "%PLF_HOME%\addons\addons-man
 
 :execCmd
 set JAVA_OPTS=-Dplf.home=%PLF_HOME%
```

Comment:
like for linux, we just need to insert the new variable `EXO_AM_OPTS` value in `JAVA_OPTS`

```suggestion
set JAVA_OPTS=-Dplf.home=%PLF_HOME% %EXO_AM_OPTS%
```